tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -471,7 +473,7 @@ def wait_job_running(self, job_id):
 
     def get_node_info(self, nodename):
         """Get node info."""
-        return self._remote_command_executor.run_remote_command("scontrol show nodes {0}".format(nodename))
+        return self._remote_command_executor.run_remote_command("scontrol show nodes {0}".format(nodename)).stdout
```

Comment:
Is this method even used in the code?

Also, we might implement it the same way we implemented `get_jobs_info()` (a bit above in the code):

```python
    def get_job_info(self, job_id, field=None):
        """Return job details from slurm. If field is provided, only the field is returned"""
        result = self._remote_command_executor.run_remote_command("scontrol show jobs -o {0}".format(job_id)).stdout
        if field is not None:
            match = re.search(rf"(\s{field})=(\S*)", result)
            return match.group(2)
        return 
```

Reply:
the method is used in the code. We don't use field=None here so lets consider add it when it is needed in the future

Reply:
Are you sure it's being used? I only see references to a `get_nodes_info` (with the s)  method used in node, but nothing about `get_node_info`. Anyway, it's not too important.

Reply:
yes I am sure. get_node_info is used, see code change in my PR: tests/integration-tests/tests/common/hit_common.py